### ISSUE TITLE
[core-kit] need to update `thin-provisioning-tools` for `sys-fs/lvm2`

### ISSUE THREAD

**Issue by cuantar** [macaroni-os/mark-issues#2]
`USE=thin` enables a dep on `thin-provisioning-tools` and is used by Docker.

```
skipped: (sys-fs/lvm2-2.03.25:0/0::core-kit, ebuild scheduled for merge) (see unsatisfied dependency below)

!!! All ebuilds that could satisfy ">=sys-block/thin-provisioning-tools-1.0.6" have been masked.
!!! One of the following masked packages is required to complete your request:
- sys-block/thin-provisioning-tools-9999::core-kit (masked by: missing keyword)

(dependency required by "sys-fs/lvm2-2.03.25::core-kit[lvm,thin]" [ebuild])
For more information, see the MASKED PACKAGES section in the emerge
man page or refer to the Gentoo Handbook.
```

```
[I] sys-block/thin-provisioning-tools
     Available versions:  0.7.0 (~)0.7.6 (~)0.8.3 (~)0.8.5 **9999*l {static test}
     Installed versions:  0.8.5(19:13:57 23/09/23)(-static -test)
     Homepage:            https://github.com/jthornber/thin-provisioning-tools
     Description:         A suite of tools for thin provisioning on Linux
     ```

**Comment by geaaru** [comment]
https://github.com/macaroni-os/macaroni-funtoo/blob/f155c9eb3cf080f4656709954837f212e32670b4/packages/atoms/base-tools2/sys-fs/lvm2/build.yaml#L10

**Comment by cepelinas9000** [comment]
That package has catch: code is rewritten to rust  and large huge depedencies (i didn't follow recently how funtoo was handling rust package dependencies)

As for working example with gentoo eclasses amalgamation that package is in my overlay: https://codeberg.org/gtautvis/tautvis-public-overlay (just don'y recommend blindly add - i screwed with gentoo eclasses...)

**Comment by cuantar** [comment]
Fixed in macaroni-os/kit-fixups#60.
